src/corewlan.cs

```diff
@@ -544,4 +544,34 @@ interface CWNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopyi
 	interface CWMutableNetworkProfile : NSCoding, NSSecureCoding, NSCopying, NSMutableCopying  
 	{
 	}
+
+	[BaseType (typeof (NSObject))]
```

Comment:
Since the header defines:
NS_CLASS_AVAILABLE_MAC(10_10)

We need to add [Mac (10,10)] to the top.